radixdlt/src/main/java/com/radixdlt/consensus/ConsensusException.java

```diff
@@ -1,28 +1,35 @@
-/*
- *  (C) Copyright 2020 Radix DLT Ltd
- *
- *  Radix DLT Ltd licenses this file to you under the Apache License,
- *  Version 2.0 (the "License"); you may not use this file except in
- *  compliance with the License.  You may obtain a copy of the
- *  License at
- *
- *   http://www.apache.org/licenses/LICENSE-2.0
- *
- *  Unless required by applicable law or agreed to in writing,
- *  software distributed under the License is distributed on an
- *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
- *  either express or implied.  See the License for the specific
- *  language governing permissions and limitations under the License.
- */
-
-package com.radixdlt.consensus;
-
-/**
- * Manages safety of the protocol.
- * TODO: Add storage of private key of node here
- */
-public final class SafetyRules {
-	public Vote vote(Vertex vertex) {
-		return new Vote(vertex.getRound(), vertex.hashCode());
-	}
-}
+/*
+ *  (C) Copyright 2020 Radix DLT Ltd
+ *
+ *  Radix DLT Ltd licenses this file to you under the Apache License,
+ *  Version 2.0 (the "License"); you may not use this file except in
+ *  compliance with the License.  You may obtain a copy of the
+ *  License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *  Unless required by applicable law or agreed to in writing,
+ *  software distributed under the License is distributed on an
+ *  "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND,
+ *  either express or implied.  See the License for the specific
+ *  language governing permissions and limitations under the License.
+ */
+
+package com.radixdlt.consensus;
+
+/**
+ * Generic runtime exception in a {@link Consensus}
+ */
+public class ConsensusException extends RuntimeException {

```

Comment:
What? SafetyRules -> ConsensusException doesn't make sense to me

Reply:
Unfortunately git is not always great at working out things that were renamed versus new code :)